hydro_cli/examples/simple/main.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    let mut input = String::new();
+    std::io::stdin().read_line(&mut input).unwrap();
+    let trimmed = input.trim();
+    println!("Hello, world, I got {trimmed}!");
+}
```

Comment:
TODO

Reply:
Yeah, I need to write logic to parse the networking information that is passed through stdin and use that to set up a simple HF spinner.